puma/hlplots/results.py

```diff
@@ -0,0 +1,340 @@
+"""Results module for high level API."""
+from puma import Histogram, HistogramPlot, Roc, RocPlot, VarVsEff, VarVsEffPlot
+from puma.metrics import calc_rej
+from puma.utils import get_good_linestyles, global_config, logger
+
+
+class ResultsBase:
+    """Base class handling plotting of tagger results."""
+
+    def __init__(self) -> None:
+        self.taggers = []
+        self.model_names = []
```

Comment:
Could these two just be a single dict?

Reply:
I kept them as 2 lists to maintain the order in which the taggers were added to the results

Reply:
I now also changed their names to start with a `_` to make clear those are internal values

Reply:
I believe the ordering of keys in a standard python `dict` is preserved in the order in which they are added as of python 3.7. There is also `OrderedDict` which achieves the same